cmd/flipt/import.go

```diff
@@ -34,6 +35,12 @@ func newImportCommand() *cobra.Command {
 		false,
 		"drop database before import",
 	)
+	cmd.Flags().BoolVar(
+		&importCmd.newFlagsOnly,
+		"append",
```

Comment:
im not sure which is more clear between:

- append
- skip-existing
- (something else?)

cc @GeorgeMac

Reply:
This two is definetely more clear than `--append`:
* `--skip-existing`
* `--insert-only`

If in the future flag `--upsert` will be added, `--insert-only` has pretty good assotiation with it.

Reply:
But `--skip-existing` is more clear for user. Let's rename it as flag and inside

Reply:
Well, I've renamed it to `--skip-existing` flag and `skipExisting` var name. Please have a look.

Reply:
If you prefer another name, please just write a comment - I'll change it

Reply:
`--skip-existing` makes sense to me 👍